features/large_map.feature

```diff
@@ -0,0 +1,38 @@
+Feature: As a Customer
+  in order to see restaurants in my area
+  I need to see a map and list with local restaurants.
+
+Background:
+  Given the following users exist
+  | name   | email         | role  |
+  | Anna   | anna@food.se  | owner |
+  | Janne  | janne@food.se | owner |
+  And "Anna" owns the following restaurant
+  | name | description | town |
+  | McD  | Nice food   | Home |
+  And "Janne" owns the following restaurant
+  | name | description | town |
+  | McF  | Nice food   | Gone |
+  And I am on the "index" page
```

Comment:
I don't understand those backgrounds. Why do we care whose restaurants these are?